src/exec/Main.hs

```diff
@@ -523,6 +526,20 @@ runByron tracer byronOptions genesisConfig blockConfig updateConfig nodeConfig e
     (\tbuilder -> (Text.pack "main.client", Monitoring.Info, tbuilder))
     tracer
 
+-- | For use by the Shelley server/client error policy tracers.
+defineSeverity :: ErrorPolicyTrace -> Monitoring.Severity
```

Comment:
Copied from a typeclass instance in `cardano-explorer` @coot

Reply:
Which was copied from `cardano-node` - we should move the orphans to `ouroboros-neteork`.